pkg/secrets/push_to_file/secrets_groups_test.go

```diff
@@ -0,0 +1,178 @@
+package secrets_groups
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type extractSecretsGroupsTestCase struct {
+	description string
+	contents    map[string]string
+	assert      func(t *testing.T, result SecretsGroups, err error)
+}
+
+func assertValidSecretsGroups(expected SecretsGroups) func(*testing.T, SecretsGroups, error) {
```

Comment:
Function is named `assertValidSecretsGroup`, but maybe it should be something like `assertExpectedSecretsGroup`? i.e. it's checking that the results are expected, not just that they're valid?

Reply:
Done